infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/project/OpenShiftProject.java

```diff
@@ -130,27 +126,11 @@ void prepare(
       }
 
       if (initWithCheServerSa) {
-        OpenShiftClient openshiftClient = cheServerOpenshiftClientFactory.createOC();
-        create(projectName, openshiftClient);
-        waitDefaultServiceAccount(projectName, openshiftClient);
-        openshiftClient
-            .roleBindings()
-            .inNamespace(projectName)
-            .createOrReplace(
-                new RoleBindingBuilder()
-                    .withNewMetadata()
-                    .withName("admin")
-                    .endMetadata()
-                    .addToUserNames(osClient.currentUser().getMetadata().getName())
-                    .withNewRoleRef()
-                    .withApiVersion("rbac.authorization.k8s.io")
-                    .withKind("RoleBinding")
-                    .withName("admin")
-                    .endRoleRef()
-                    .build());
-      } else {
```

Comment:
This looks like the main change relevant to the project creation and role provisioning, but I'm not sure if this what we would like to achieve. e.g. by design user gets admin permissions in the namespace and this should be just fine.

Reply:
That's legacy code.
Basically I make  OpenShift Infrastructure project creation codebase similar to Kuberntes infrastructure.